ee/libvux/src/vuhw.c

```diff
@@ -18,13 +18,23 @@ void Vu0IdMatrix(VU_MATRIX *m)
 void Vu0ResetMatrix(VU_MATRIX *m)
 {
 	asm __volatile__(
+#if __GNUC__ > 3
+	"vmr32.xyzw  $vf18, $vf0    \n"
+	"sqc2        $vf0, 0x30(%0)   \n"
+	"vmr32.xyzw  $vf17, $vf18   \n"
+	"sqc2        $vf18, 0x20(%0)    \n"
+	"vmr32.xyzw  $vf16, $vf17   \n"
+	"sqc2        $vf17, 0x10(%0)    \n"
+	"sqc2        $vf16, 0x00(%0)    \n"
+#else
```

Comment:
tabs/spaces/aligning does not look right.
The same for all other assembly code.